iib/workers/tasks/build.py

```diff
@@ -358,6 +391,22 @@ def _opm_index_add(
         ``overwrite_from_index``. The format of the token must be in the format "user:password".
     :raises IIBError: if the ``opm index add`` command fails.
     """
+    index_image_bundles = _serve_index_db(from_index, base_dir)
```

Comment:
This should probably be a separate helper method.

Reply:
I have separated the code into more methods.